src/js/SignInUI.js

```diff
@@ -23,10 +25,17 @@ export default class SignIn {
       AuthAPI.login({
         identifier: user.value.trim(),
         password: pass.value.trim(),
-      }).catch((e) => {
-        const errorMsg = e.response.data.message[0].messages[0].message;
-        SignIn.errorLog(errorMsg, passError);
-      });
+      })
+        .then(()=> {
+          loginModal.classList.remove('active');
+        })
+        .then(()=> {
+          new UserUI().init(true);
```

Comment:
```suggestion
        .then(()=> {
          loginModal.classList.remove('active');
          new UserUI().init(true);
```

you can group into one